beesdoo_product/views/beesdoo_product.xml

```diff
@@ -45,6 +45,10 @@
                             <field name="label_last_printed"/>
                         </group>
                     </group>
+                    <group>
+                        <field name="fds_label"/>
+                        <field name="hazard_label"/>
```

Comment:
add widget="selection" or options="no_create" to avoid the user to create label on the fly from here.

Reply:
Thanks for the tip (options="no_create" doesn't seem to work here though)